deployment/Triton/client/client.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Hi @wyli @ericspod , I am not very sure whether this copyright matches MONAI copyright, could you please help double confirm it?

Thanks.

Reply:
It got missed in a commit, I've added the headers.
# Copyright 2020 - 2021 MONAI Consortium
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#     http://www.apache.org/licenses/LICENSE-2.0
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

Reply:
It got missed in a commit, I've added the headers from:
https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#checking-the-coding-style